tests/lib.rs

```diff
@@ -1,2 +1,4 @@
+#![feature(once_cell)]
```

Comment:
What does it do?

Reply:
I was experimenting with Lazy. I'll remove it, as that didn't get me anywhere.